src/Symfony/Component/Yaml/composer.json

```
@@ -23,10 +23,10 @@
         "symfony/polyfill-php81": "^1.22"
     },
     "require-dev": {
-        "symfony/console": "^4.4|^5.0|^6.0"
+        "symfony/console": "^5.3"
```

`^6.0` should still be allowed, right?

```suggestion
        "symfony/console": "^5.3|^6.0"
```